api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -46,6 +50,8 @@
 	
 	public static final String IDENTIFIER = "identifier";
 	
+	public static final String DIAGNOSTIC_REPORT_CATEGORY = "LAB";
```

Comment:
This is not a good name for this constant. It should probably be something like `DIAGNOSTIC_REPORT_CATEGORY_LAB`.